test/scripts/stop-fab-network.sh

```diff
@@ -10,7 +10,9 @@ if [ -n  "$(docker ps -a | egrep 'fabsdkgo|fabric-tools|fabric-ccenv' | awk '{pr
 echo "removing containers ..."
     docker rm $(docker ps -a | egrep 'fabsdkgo|fabric-tools|fabric-ccenv' | awk '{print $1}')
 fi
-if [ -n  "$(docker images | grep 'fabric' | awk '{print $3}')" ]; then
+
+# delete only temporary chaincode images (keep other fabric docker images for retest)
+if [ -n  "$(docker images | grep 'fabsdkgo' | awk '{print $3}')" ]; then
 echo "removing images ..."
-    docker rmi $(docker images | grep 'fabric' | awk '{print $3}')
+    docker rmi $(docker images | grep 'fabsdkgo' | awk '{print $3}')
```

Comment:
Are you sure the images are named fabsdkgo? The ones I had when running the tests where fabric-. That's why the tests cleanup where failing for me. Is this a configurable param?

Reply:
I do not want to remove all stable fabric images, which may have been downloaded for other works. But, "fabsdkgo" are chaincode images created by this test, and thus must be cleaned up.

Reply:
Ok, I'll approve but tests should cleanup after themselves and these ones leave all those images behind. I agree that it might have been downloaded by something else, but maybe we should have a dedicated target that does not cleanup or something...